HDPS/src/LinkedData.cpp

```diff
@@ -192,12 +170,13 @@ LinkedData::LinkedData() :
 }
 
 LinkedData::LinkedData(const Dataset<DatasetImpl>& sourceDataSet, const Dataset<DatasetImpl>& targetDataSet) :
-    Serializable("LinkedData"),
-    _sourceDataSet(sourceDataSet),
-    _targetDataSet(targetDataSet)
+    LinkedData()
 {
-    Q_ASSERT(_sourceDataSet.isValid());
-    Q_ASSERT(_targetDataSet.isValid());
+    Q_ASSERT(sourceDataSet.isValid());
```

Comment:
I want to keep the initializations for the `_sourceDataSet` and `_targetDataSet` in the constructor initialization list as before. The assignment is very lightweight for the `Dataset<DatasetImpl>` (merely copying a pointer and a string). I currently do not see problems with the original approach.